src/lib/i18next.js

```diff
@@ -6,44 +6,36 @@
 
 import i18n from 'i18next';
 
-import TRANSLATIONS_DE from './translations/de/translations.json';
-import TRANSLATIONS_EN from './translations/en/translations.json';
-import TRANSLATIONS_TR from './translations/tr/translations.json';
-
 import LanguageDetector from 'i18next-browser-languagedetector';
+const { languages } = require('./package').config;
 
 const options = {
-  fallbackLng: 'en',
-  returnEmptyString: false, // fallback keys
-    debug: true,
-    resources: {
-      en: {
-        translation: TRANSLATIONS_EN,
-      },
-      de: {
-        translation: TRANSLATIONS_DE,
-      },
-      tr: {
-        translation: TRANSLATIONS_TR,
-      },
-    },
-    // specify language detection order
-    detection: {
-      order: ['htmlTag'],
-      // cache user language off
-      caches: [],
-    }
-}
+  fallbackLng: 'en', // fallback keys
+  returnEmptyString: false,
+  debug: true,
+  resources: languages.reduce(async (obj, lang) => {
```

Comment:
good job - doing this :)

Reply:
I had to delete this as the dynamic imports would require to publish the json files with the translations and a request would be made on runtime...